app/components/Channels/ChannelData.js

```diff
@@ -24,7 +24,7 @@ const ChannelData = ({ channel, currencyName, networkInfo, viewMode, ...rest })
     csv_delay,
     num_updates,
   } = channel
-  const [fundingTxid] = channel_point.split(':')
+  const [fundingTxid] = channel_point ? channel_point.split(':') : []
```

Comment:
When `channel_point ` is null or undef `fundingTxid` will be `undefined`. Is this ok ? I see that we use down below to open block explorer and show text.

Reply:
Yes, that's fine because we filter out the null values here

https://github.com/LN-Zap/zap-desktop/blob/67ffbd5733b5c15837c351ce167159cd695d1019/app/components/Channels/ChannelData.js#L130

Reply:
Ok makes sense. Ideally i'd use `Boolean(channel[id])` instead of `channel[id] != null` since latter implies implicit conversion undefined to null.

Reply:
`Boolean(channel[id])` would filter out things where the value was `0`, which we don't want here. We only want to filter out things that are `null` or `undefined`

Reply:
I've refactored this as per slack convo, and added some comments.